benches/bench.rs

```diff
@@ -18,7 +18,7 @@ macro_rules! bench_num {
                 let buf = $data;
                 b.iter(|| {
                     for _ in 0..NITER {
-                        bb(BigEndian::$read(&buf, $bytes));
+                        bb(BigEndian::$read(&buf, bb($bytes)));
```

Comment:
Not sure whether the `$bytes` input should be wrapped in `black_box`, but I noticed it has a huge impact on performance, so I figured I'd mention it

Reply:
Hmmm I don't think so. But I'm honestly not sure. I'd have to see a minimal reproducible example and compare codegen I think.

Reply:
I think I would defer to the status quo for now.